### ISSUE TITLE
"import admit" produces a WARNING

### ISSUE THREAD

**Issue by yeunga** [astroumd/admit#36]
ADMIT produces a WARNING when "import admit" is executed inside a CASA session. The following is a capture of the session output. This WARNING appears from CASA V5.1.x to V5.6.1-8.
---
# source /opt/admit/admit_start.sh
[root@a74bdfb47546 /]# admit
ADMIT        = /opt/admit
    version  = 1.0.8.2
CASA_PATH   = /opt/casa-pipeline-release-5.6.1-8.el7 linux admit a74bdfb47546
CASA_ROOT    = /opt/casa-pipeline-release-5.6.1-8.el7
    prefix   = /opt/casa-pipeline-release-5.6.1-8.el7
    version  = release-pipeline-5.6.1-8
    revision = 1
[root@a74bdfb47546 /]# casa

=========================================
The start-up time of CASA may vary
depending on whether the shared libraries
are cached or not.
=========================================

IPython 5.1.0 -- An enhanced Interactive Python.

PIPELINE CASA 5.6.1-8   -- Common Astronomy Software Applications

Creating a new telemetry file
Telemetry initialized. Telemetry will send anonymized usage statistics to NRAO.
You can disable telemetry by adding the following line to your ~/.casarc file:
EnableTelemetry: False
--> CrashReporter initialized.
Enter doc('start') for help getting started with CASA...
Using matplotlib backend: TkAgg

CASA <1>: import admit
WARNING: No ASAP; ASAPLineFinder cannot function.

CASA <2>:
---

**Comment by teuben** [comment]
Indeed, a few versions ago CASA didn't ship the ASAP package anymore. This was used for single dish mostly, and has been superseded by the "sd" tools. Too bad, as they did have a a nice line finder, which could be optionally used.    In the next release we will silence the warning one way or the other. Until then I'll leave the issue open.
